docker-compose-generator/docker-fragments/opt-add-bwt.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
+
+services:
+  bwt:
+    image: shesek/bwt
```

Comment:
can you put a specific tag on this?

Reply:
There's also a `shesek/bwt:electrum` (or `shesek/bwt:x.y.z-electrum`) image that you could use to enable just the Electrum RPC server, without the HTTP API. This is more lightweight, avoids a bunch of big dependencies and one database index. Maybe let the user pick somehow?